### PR TITLE
TOMEE-2698: Translate to spanish simple-rest

### PR DESCRIPTION
**Pull request by breakponchito**
Link to Jira: https://issues.apache.org/jira/browse/TOMEE-2698

### CODE REVIEW

File: examples/simple-rest/README_es.adoc [hunk 1]
```diff
@@ -0,0 +1,140 @@
+:index-group: REST
+:jbake-type: page
+:jbake-status: status=published
+
+= REST Simple
+
+Definir un servicio Rest es muy fácil, simplemente necesitamos agregar la anotación ``@Path`` en la clase y después definir los métodos HTTP que vamos a usar (``@GET``, ``@POST``, …).
```
Comment (emecas) [inline examples/simple-rest/README_es.adoc]: I did not know you can use indistinctly single quote `  or 2 single quotes ``, interesting

File: examples/simple-rest/README_es.adoc [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+= El Código
+
+== El servicio REST: ``@Path``, ``@GET``, ``@POST``
+
+Aquí tenemos un REST simple, anotamos la clase con ``@Path("/greeting")`` para indicar la ruta correspondiente a la clase ``GreetingService``. Definimos ``message()`` como ``@GET`` y ``lowerCase()`` como ``@POST`` para la ruta ``/greeting``. Pronto tenemos un servicio, muy simple ¿verdad?
+
```
Comment (emecas) [inline examples/simple-rest/README_es.adoc]: Here you may include this label: [source,java] as well as line 47

File: examples/simple-rest/README_es.adoc [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+        final String message = WebClient.create("http://localhost:4204").path("/GreetingServiceTest/greeting/").post("Hi REST!", String.class);
+        assertEquals("hi rest!", message);
+    }
+}
+....
+
+== Ejecución
+
+Ejecutar la aplicación es muy simple. En el directorio ``simple-rest`` ejecuta:
+
```
Comment (emecas) [inline examples/simple-rest/README_es.adoc]: I have seen the label: [source,bash] , I think you can use this one as well in line: 94

File: examples/simple-rest/README_es.adoc (hunk 2 above)
Comment (breakponchito) [reply examples/simple-rest/README_es.adoc]: Excellent, let me do that

File: examples/simple-rest/README_es.adoc (hunk 3 above)
Comment (breakponchito) [reply examples/simple-rest/README_es.adoc]: excellent, let me do that

File: examples/simple-rest/README_es.adoc (hunk 2 above)
Comment (breakponchito) [reply examples/simple-rest/README_es.adoc]: done!!

File: examples/simple-rest/README_es.adoc (hunk 3 above)
Comment (breakponchito) [reply examples/simple-rest/README_es.adoc]: done!!

**Pull request merged**